code/datums/uplink/devices and tools.dm

```diff
@@ -85,9 +70,19 @@
 
 /datum/uplink_item/item/tools/space_suit
 	name = "Mercenary Voidsuit"
-	item_cost = 6
+	item_cost = 12
 	path = /obj/item/storage/box/syndie_kit/space
 
+/datum/uplink_item/item/tools/mercarmor
+	name = "Mercenary Combat Armor"
+	item_cost = 6
+	path = /obj/item/storage/box/syndie_kit/mercarmor
```

Comment:
This and the mercenary vest could use a `desc` to explain what's actually inside (heavy combat armor = vest+helmet)

Reply:
It has one now, and I changed the screenshot to the new description. It starts off the same as the other one, but it is different.